renderdoc/driver/gl/gl_pixelhistory.cpp

```diff
@@ -83,6 +186,12 @@ bool PixelHistoryDestroyResources(WrappedOpenGL *driver, const GLPixelHistoryRes
   driver->glDeleteTextures(1, &resources.colorImage);
   driver->glDeleteTextures(1, &resources.dsImage);
   driver->glDeleteFramebuffers(1, &resources.frameBuffer);
+  driver->glDeleteShader(resources.primitiveIdFragmentShader);
```

Comment:
we forgot to destroy the texture/framebuffer we created. Won't matter if we delete it from the code though